src/crate/client/sqlalchemy/tests/bulk_test.py

```diff
@@ -74,4 +74,4 @@ def test_bulk_save(self):
             ('Banshee', 26),
             ('Callisto', 37)
         )
-        self.assertEqual(expected_bulk_args, bulk_args)
+        self.assertSequenceEqual(expected_bulk_args, bulk_args)
```

Comment:
With SA<2.0, `bulk_args` is a `tuple`, so `expected_bulk_args` was written as such. Now, with SA>=2.0, it is a `list`. Because this only concerns the call arguments to `cursor.executemany()`, which is invoked by SQLAlchemy, I think we do not need to put any kind of admonition notice about a breaking change into the changelog, as it looks like it will not concern the user of the library.

Please let me know if you think differently.